proxy-server/util/settings.go

```diff
@@ -46,8 +46,8 @@ func LoadSettings(settingsFile string) (*Settings, error) {
 		ProxyConfigMaps:           proxyConfigMaps,
 		ProxyPort:                 8080,
 		ProxyURLs:                 proxyURLs,
-		RefreshTokenExpirationStr: "3600s",
-		SaturnTokenExpirationStr:  "86400s",
+		RefreshTokenExpirationStr: "86400s",
+		SaturnTokenExpirationStr:  "3600s",
```

Comment:
Curious why these swapped? Were they incorrect before?

Reply:
Yeah, bad defaults. Did a big refactor to move all of the config to a YAML file instead of env vars (sound familiar?), and apparently swapped these.

NBD though, because we were overriding them with the correct values anyway from Atlas.